Source/Tagging.H

```diff
@@ -140,7 +141,8 @@ struct EmptyProbTagStruct
     const amrex::GpuArray<amrex::Real, AMREX_SPACEDIM> /*dx*/,
     const amrex::GpuArray<amrex::Real, AMREX_SPACEDIM> /*prob_lo*/,
     const amrex::Real /*time*/,
-    const int /*level*/) noexcept
+    const int /*level*/,
+    ProbParmDevice const& d_prob_parm_device) noexcept
```

Comment:
can you just add `/* ... */` around `d_prob_parm_device`? Here and in the two other prob.H files edited.

Reply:
I do think it would be ideal to at least have the same syntax for setting runtime refinement criteria between the two codes. PeleLM's is definitely more elegant and flexible. But the existing PeleC approach does seem to enable all types of refinement that actually get used, so it seems fine for now.

In any event, the target of this pull request is somewhat separate from that discussion.

Reply:
Absolutely.

Reply:
@baperry2 Yes, it is separate.